src/org/zaproxy/zap/extension/spider/SpiderPanel.java

```diff
@@ -160,6 +163,9 @@
 	/** The found count value label. */
 	private JLabel foundCountValueLabel;
 	
+	private static ZapTableExportButton exportUrlsButton = null;
+	private static ZapTableExportButton exportMessagesButton = null;

```

Comment:
I did start that way but had no luck, however that was before I switched away from instanced buttons.....will fight with both a bit more I guess